### ISSUE TITLE
[A11y]

### ISSUE THREAD

**Issue by angeloko23** [IBMa/equal-access#149]
## Environment

> Operating system:

> Browser:

> Automated testing tool and ruleset:

> Assistive technology used to verify:

## Detailed description

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

- Screenshots or code
- Notes

**Comment by tombrunet** [comment]
@angeloko23 I do not see any information listed here as to what this issue is about. I'm closing this one. If you have something you wished to report, please open a new issue.
